SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -599,6 +599,7 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
     if (Coord_i[1] < EPS) return;
 
     AD::SetPreaccIn(Coord_i[1]);
+    AD::SetPreaccIn(V_i[2]);
```

Comment:
And if I could ask one more thing, please replace the hardcoded "2" with
```suggestion
    AD::SetPreaccIn(V_i[idx.Velocity() + 1]);
```
And similarly for uses of `PrimVar_Grad_i` in line 614